beacon_node/execution_layer/src/lib.rs

```diff
@@ -228,6 +228,7 @@ impl<T: EthSpec> ExecutionLayer<T> {
                 .map_err(Error::InvalidJWTSecret)
         } else {
             // Create a new file and write a randomly generated secret to it if file does not exist
+            warn!(log, "No JWT found given path. Generating" ; "path" =>default_datadir.join(&secret_file).as_path().to_str().unwrap());
```

Comment:
The `secret_file` is already a full path, so no need to join it to the datadir.

We can also display a nice form of the path using `%` (to use `Display`) and `.display()` (to sanitize the path and make it implement `Display`).

```suggestion
            warn!(log, "No JWT found on disk. Generating"; "path" => %secret_file.display());
```